pkg/trace/info/info_test.go

```diff
@@ -177,6 +178,38 @@ func TestInfo(t *testing.T) {
 	assert.Equal(expectedInfoString, info)
 }
 
+func TestProbabilisticSampler(t *testing.T) {
+	assert := assert.New(t)
+	conf := testInit(t)
+	assert.NotNil(conf)
+
+	server := testServer(t, "./testdata/psp.json")
+	assert.NotNil(server)
+	defer server.Close()
+
+	url, err := url.Parse(server.URL)
+	assert.NotNil(url)
+	assert.NoError(err)
+
+	hostPort := strings.Split(url.Host, ":")
+	assert.Equal(2, len(hostPort))
+	port, err := strconv.Atoi(hostPort[1])
+	assert.NoError(err)
+	conf.DebugServerPort = port
+
+	var buf bytes.Buffer
+	err = Info(&buf, conf)
+	assert.NoError(err)
+	info := buf.String()
+	assert.NotEmpty(info)
+	t.Logf("Info:\n%s\n", info)
+	expectedInfo, err := os.ReadFile("./testdata/psp.info")
+	re := regexp.MustCompile(`\r\n`)
+	expectedInfoString := re.ReplaceAllString(string(expectedInfo), "\n")
```

Comment:
I'm likely missing something, but why don't we just ensure `psp.info` contains `\n` line breaks?

Reply:
I _think_ this is to deal with inconsistencies running this test on Windows? (to be honest here I copied this logic from a different test and didn't think super critically about this line)